src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Inputs/BitTextFieldTests.cs

```diff
@@ -225,43 +192,44 @@ public void BitTextFieldMustRespondToTheFocusEvent(bool isEnabled, bool isMultiL
             DataRow(false, false),
             DataRow(false, true),
         ]
-        public void BitTextFieldMustRespondToTheKeyEvent(bool isEnabled, bool isMultiLine)
+        public void BitTextFieldMustRespondToTheKeyEvent(bool isEnabled, bool IsMultiline)
         {
             var component = RenderComponent<BitTextFieldTest>(
                 parameters =>
                 {
                     parameters.Add(p => p.IsEnabled, isEnabled);
-                    parameters.Add(p => p.IsMultiLine, isMultiLine);
+                    parameters.Add(p => p.IsMultiLine, IsMultiline);
                 });
 
-            var bitTextField = isMultiLine ? component.Find(".bit-txt textarea") : component.Find(".bit-txt input");
+            var bitTextField = IsMultiline ? component.Find(".bit-txt textarea") : component.Find(".bit-txt input");
 
             bitTextField.KeyDown("down");
-            Assert.AreEqual(isEnabled ? 1 : 0, component.Instance.CurrentCount);
+            Assert.AreEqual(isEnabled ? 1 : 0, component.Instance.KeyDownedValue);
 
             bitTextField.KeyUp("up");
-            Assert.AreEqual(isEnabled ? 2 : 0, component.Instance.CurrentCount);
+            Assert.AreEqual(isEnabled ? 1 : 0, component.Instance.KeyUpedValue);
         }
 
-        [DataTestMethod,
-           DataRow(true, false),
-           DataRow(true, true),
-           DataRow(false, false),
-           DataRow(false, true),
-       ]
-        public void BitTextFieldMustRespondToTheChangeEvent(bool isEnabled, bool isMultiLine)
-        {
-            var component = RenderComponent<BitTextFieldTest>(
-                parameters =>
-                {
-                    parameters.Add(p => p.IsEnabled, isEnabled);
-                    parameters.Add(p => p.IsMultiLine, isMultiLine);
-                });
-
-            var bitTextField = isMultiLine ? component.Find(".bit-txt textarea") : component.Find(".bit-txt input");
-
-            bitTextField.Change("change");
-            Assert.AreEqual(isEnabled ? 1 : 0, component.Instance.CurrentCount);
-        }
+       // [DataTestMethod,
+       //    DataRow(true, false),
+       //    DataRow(true, true),
+       //    DataRow(false, false),
+       //    DataRow(false, true),
+       //]
+       // public void BitTextFieldMustRespondToTheChangeEvent(bool isEnabled, bool IsMultiline)
+       // {
+       //     var component = RenderComponent<BitTextFieldTest>(
+       //         parameters =>
+       //         {
+       //             parameters.Add(p => p.IsEnabled, isEnabled);
+       //             parameters.Add(p => p.IsMultiLine, IsMultiline);
+       //         });
+
+       //     var bitTextField = IsMultiline ? component.Find(".bit-txt textarea") : component.Find(".bit-txt input");
+
+       //     bitTextField.KeyDown("a");
+
+       //     Assert.AreEqual(isEnabled ? 1 : 0, component.Instance.CurrentCount);
+       // }
```

Comment:
again, please only comment out the assertion lines, not the whole test method, just like the ones done before.

Reply:
done